### PR TITLE
[chip,dv] update sampling clock in sysrst_ctrl test

### PR DESCRIPTION
**Pull request by jdonjdon**
chip_sw_sysrst_ctrl_ec_rst_l uses fixed 5us clock to sample `ec_rst_l`. 
Since this clock can be drifted +/- 3%, 
constant clock can cause reset period measurement failure.
Replace this sampling clock to real one.

### CODE REVIEW

**Review by a-will — APPROVED** [review]

File: hw/top_earlgrey/dv/env/seq_lib/chip_sw_sysrst_ctrl_ec_rst_l_vseq.sv
Comment (marnovandermaas) [inline hw/top_earlgrey/dv/env/seq_lib/chip_sw_sysrst_ctrl_ec_rst_l_vseq.sv:85]: Maybe change this comment to be a 1 cycle delay?

Comment (jdonjdon) [reply hw/top_earlgrey/dv/env/seq_lib/chip_sw_sysrst_ctrl_ec_rst_l_vseq.sv]: sure

Comment (marnovandermaas) [reply hw/top_earlgrey/dv/env/seq_lib/chip_sw_sysrst_ctrl_ec_rst_l_vseq.sv]: Thanks

**Review by marnovandermaas — APPROVED** [review]
LGTM. Let's wait for CI to pass before merging.

**Pull request merged**
